sdk/python/kfp/compiler/_container_builder.py

```diff
@@ -18,19 +18,37 @@
 import os
 import uuid
 
+SERVICEACCOUNT_NAMESPACE = '/var/run/secrets/kubernetes.io/serviceaccount/namespace'
+GCS_STAGING_BUCKET_DEFAULT = 'gs://kfp_container_build_staging'
```

Comment:
Should this name need to be namespaced by project id? Otherwise, how does it work across projects?

how about calling it <project_id>_kfp_staging? Other KFP use case which requires a staging gcs bucket can reuse the same default bucket but with different subpaths.

Reply:
If you decide to go with this approach, could you create a subdirectory under staging dir called kaniko_artifacts or other name?

Reply:
Good catch. I thought the bucket namespace is within the project.
Will update

Reply:
The GCS bucket names must be globally unique.

Reply:
I'm planning to go with gs://project_id/kfp_container_build_staging/, WDYT?

Reply:
Done. PTAL.